tests/test_layers.py

```diff
@@ -89,22 +112,24 @@ def adm_area(mocker, multipolygon):
     adm_area.adm_name = "Timor-Leste"
     return adm_area
 
+@pytest.fixture
+def tags():
```

Comment:
I’d call it something like osm_hospital_tags because I think it would improve readability

Reply:
Good one, it was a bit too generic before.